api/python/provisioner/commands/check.py

```diff
@@ -621,9 +799,16 @@ def run(self, check_name: cfg.Checks = None,
         :return:
         """
         res: CheckResult = CheckResult()
+        check_lists = None
+
+        if check_name is None or check_name == cfg.GroupChecks.ALL.value:
+            check_lists = cfg.CHECKS
+
+        if not check_lists and check_name in cfg.GROUP_CHECKS:
+            check_lists = getattr(cfg, check_name.strip().upper())
```

Comment:
We again are changing the base logic of `run` method. We need to allow a user to run the `check` command by name of group checks as well as a single check. E.g.
```
$ provisioner check pre_checks
```
will work in current implementation but
```
$ provisioner check connectivity
```
won't work.

Reply:
@dmitrydb This checks for both singular checks as well as group checks. 

`[root@smc67-m01 cortx-prvsnr]# provisioner check connectivity
{
    "connectivity": [
        "srvnode-1: passed",
        "srvnode-2: passed"
    ]
}

[root@smc67-m01 cortx-prvsnr]# provisioner check network_drivers
{
    "network_drivers": "*: passed: Network Driver Validated: ['mlnx-ofed', 'srvnode-1', 'srvnode-2']"
}

[root@smc67-m01 cortx-prvsnr]# provisioner check pre_checks
{
    "network_hca": "*: passed: HCA Presence and Ports Validated: ['mellanox', 'srvnode-1', 'srvnode-2']",
    "storage_hba": "*: passed: HBA Presence and Ports Validated: ['lsi', 'srvnode-1', 'srvnode-2']",
    "network_drivers": "*: passed: Network Driver Validated: ['mlnx-ofed', 'srvnode-1', 'srvnode-2']",
    "storage_luns": "*: passed: LUNs Accessibility, Volume Size and Mapping Validated: ['size', 'srvnode-1', 'srvnode-2']"
}`

Reply:
Sorry, Github hides from me some lines of codes that contain singular checks logic.